Practicals/prac_06/guitars.py

```diff
@@ -22,7 +23,6 @@ def main():
         print(f"{guitar_name} ({guitar_year}) : {guitar_cost} added")
         guitar_name = input("Name: ")
 
-
     largest_word_length = 0
     for guitar in guitars:
         if largest_word_length < len(guitar.name):
```

Comment:
seems like the code is not matching the practical requirement, the suggestion in practical is using the built-in enumerate() function. like  " for i, guitar in enumerate(guitars, 1): "